uswid/identity.py

```diff
@@ -442,7 +451,10 @@ def export_bytes(self) -> bytes:
         if self.lang:
             data[uSwidGlobalMap.LANG] = self.lang
         if self.tag_id:
-            data[uSwidGlobalMap.TAG_ID] = self.tag_id
+            try:
+                data[uSwidGlobalMap.TAG_ID] = uuid.UUID(hex=self.tag_id).bytes
```

Comment:
in case the self.tag_id is a UUID, how should it look like in the variable?
In my pull request it would look like this:
**aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee**
in this pull request it would have to look like this:
**aaaaaaaabbbbccccddddeeeeeeeeeeee**
but I would like it better if I can write the first one instead of the second into a json, ini, xml file (like it is currently the case).
if we would write it like the first one, we would have to write the line like this:
```suggestion
                data[uSwidGlobalMap.TAG_ID] = uuid.UUID(hex="urn:uuid:"+self.tag_id).bytes
```

Reply:
Ahh, I see, good catch. I think we actually want the dashes in the `self.tag_id` representation too -- that's much more what I expect from a json/xml/ini export PoV. Patch updated.

Reply:
I don't see any difference. I am not sure if you forgot to add the file before commiting/pushing or if I am just doing something wrong.

Reply:
I just didn't see the change...
looks good now